tutorial/chapter1.md

```diff
@@ -314,6 +310,27 @@ int main(int argc, char *argv[])
     }
 }
 ```
+Once, a quit sequence is pressed by the user join() will be called for our `imageProcessingServiceThread` and sequentially we will shutdown `fileImageStreamingService`, `imageProcessingServices`, we will remove our listeners, and, stop the event loop. 
+
+At this point, it is considered useful to understand the code of the [image_processor_service.cpp](../modules/image_processing/src/image_processor_service.cpp)  / [image_processor_service.h](../modules/image_processing/include/image_processing/image_processor_service.h) 
+
+Follows, a screenshot of the output after the execution of datamux example 1. (After, pressing the termination sequence) 
+
+![datamux example 1 output more details](../images/datamux_example1_output_more.png)
+
+## Image multiplexer example 2
+
+The main difference between the two datamux examples is that the example 2 creates 5 subscribers and 5 listeners in contrast with example 1 that creates only 1 subscriber with 5 listeners. 
+
+This actually means that example 2 is parallelized and that is visible by the output of the two examples as we will see below.
+
+- On example 1 the data received happens sequential because the subscriber shares the data with one listener at a time.
```

Comment:
This addresses lines from 327 to 333. I would put the two images first together and then add something like:

Both examples were ran for the same amount of time. As we can observe, in example 1, results show (2,2,2,1,1), this implies to again run service 1, it needed to wait for all callbacks to finish, therefore all callbacks were run sequentially. Also, the number of tasks performed in example 2 is much greater (22 vs 2) (11 vs 1). This shows one of the wonders of parallel execution, where tasks that need less resources can be executed multiple times painlessly, overall improving the throughput of the computation.

Feel free to improve it as you want, but I think some explanation like that would be good.